src/User/AbstractPolicy.php

```diff
@@ -30,8 +101,7 @@ public function subscribe(Dispatcher $events)
     }
 
     /**
-     * @param GetPermission $event
-     * @return bool|void
+     * @deprecated in favor of checkAbility
      */
     public function getPermission(GetPermission $event)
     {
```

Comment:
If this is deprecated, why don't we just call `checkAbility` from here, instead of having duplicated blocks of code.